tests/TODO.md

```diff
@@ -0,0 +1,35 @@
+### DESCRIPTION
+
+This file contains a list of test cases and notes on features which should
+be covered with automatic tests.
+
+The list is broken down by sections each of which groups tests related to some
+functional unit or feature. The format of list is as follows:
+
+#### %Name of a feature or a functional unit%:
+
+* **%Test title%**. %Short description: what this test checks%.
+  1. *%Title of test case #1%*. %Brief description of setup, action, and expected
+  result%;
+  2. *%Title of test case #2%*. `...`
+
+### TESTS
+
+#### Couple:
+
+* **DC sharing check**. When option `forbidden_dc_sharing_among_groups` is enabled,
+mastermind checks whether none of pairs of couple's groups has backends in the
```

Comment:
"whether none of pairs of couple's groups has backends in the same DC" -> "that there is no pair of couple's groups which are located in the same DC."